### ISSUE TITLE
White text in amoled mode is too bright, can we get an ability to change the "whiteness" of the text to make stuff less contrasting?

### ISSUE THREAD

**Issue by M4rtineski** [liftoff-app/liftoff#196]
### Is your feature request related to a problem? Please describe.

White text in amoled mode is too bright, can we get an ability to change the "whiteness" of the text to make stuff less contrasting?

### Describe the solution you'd like

A setting that would allow us to change how bright/white is the displayed text.

### Describe alternatives you've considered

Lower brightness/whiteness is hard coded into amoled mode.

### Additional context

Current brightness makes stuff hard to read.

**Comment by M4rtineski** [comment]
Also, I think that amoled may be a tad too dark, jerboa has great both text brightness and background blackness.

**Comment by jordan1776** [comment]
> Also, I think that amoled may be a tad too dark, jerboa has great both text brightness and background blackness.

Agreed, text color should be off white like in Jerboa or a custom setting. I think it also comes from the top and bottom bars being deep black. There should be an option for setting a secondary color so the backgrounds can be deep black, but the navigation be colored. So in total primary, secondary, and text color. Basically extending the custom theming added in 0.10.3
